docs/source/advanced/training_tricks.rst

```diff
@@ -39,6 +41,9 @@ norm <https://pytorch.org/docs/stable/nn.html#torch.nn.utils.clip_grad_norm_>`_
     # clip gradients with norm above 0.5
     trainer = Trainer(gradient_clip_val=0.5)
 
+    # clip gradients with value above 0.5
+    trainer = Trainer(gradient_clip_val=0.5, gradient_clip_algorithm='value')
```

Comment:
can we list or refer to the code what are the types, maybe just link to `GradClipAlgorithmType`

Reply:
Good idea. I changed the documentation, but don't know the way I did is correct.